README.md

```diff
@@ -82,4 +83,6 @@ Guide](https://breakthrough-energy.github.io/docs/dev/contribution_guide.html).
 
 
 
-[docs]: https://breakthrough-energy.github.io/docs/index.html
+[docs]: https://breakthrough-energy.github.io/docs/powersimdata/index.html
+[docstrings]: https://breakthrough-energy.github.io/docs/powersimdata.html#module-powersimdata
```

Comment:
I would remove #module-powersimdata.
I think https://breakthrough-energy.github.io/docs/powersimdata.html
is good enough.

Reply:
Done